### ISSUE TITLE
Filenames starting by gpr instead of grp? 

### ISSUE THREAD

**Issue by hungaborhorvath** [gap-system/gap#482]
This is a very silly observation, but it drives me crazy whenever I need to find any of these files. Some files in lib/ start by gpr instead of grp, e.g. ````gprd.gi````, ````gprdpc.gi````, ````gprdperm.gi````. I am guessing this is due to mistyping. 

Could these files be renamed to reflect their proper place in the alphabetical ordering? If these filenames are intentional, then disregard this issue, but maybe give some reasoning behind their names? Thank you.

**Comment by stevelinton** [comment]
gprd.gi is "Group PRoDuct" I think the others are similar.

**Comment by hungaborhorvath** [comment]
Ok, thanks. It still looks weird to me, especially that group is abreviated to grp, g, gp in the lib directory. Maybe these could at some point be unified? 

Anyway, if it only bothers me then there is nothing to do here. :-)

**Comment by fingolfin** [comment]
We will eventually rename the files (today we have more than 8+3 (or even 6+3 or less...) lettesr for file names avaiallable); and also use subdirectories to group things.
But we are not yet doing this, because we currently have to major branches of GAP, which we need to merge; moving files around now makes that job a lot harder.

**Comment by hulpke** [comment]
@hungaborhorvath 
The reason behind the seemingly weird naming was the 8+3 name issue (when we started, Windows 3 was the current version ….)

So group became grp, but if that pushed the length beyond 8 it became gp or even just g. gprd ended up as a portmanteau of gp and prd in the incarnation for permutation groups (gprdperm) and gprdpc and gprd were named similarly. (I just added a file gprdmat in the same naming scheme).
